src/Docker.PowerShell/Cmdlets/GetContainerNetwork.cs

```diff
@@ -1,20 +1,53 @@
 using System.Management.Automation;
 using System.Threading.Tasks;
 using Docker.DotNet.Models;
+using Docker.PowerShell.Objects;
 
 namespace Docker.PowerShell.Cmdlets
 {
-    [Cmdlet(VerbsCommon.Get, "ContainerNetwork")]
+    [Cmdlet(VerbsCommon.Get, "ContainerNet",
+            DefaultParameterSetName = CommonParameterSetNames.NetworkName)]
     [OutputType(typeof(NetworkListResponse))]
-    public class GetContainerNetwork : DkrCmdlet
+    public class GetContainerNet : DkrCmdlet
     {
+        [Parameter(ParameterSetName = CommonParameterSetNames.Default,
+            ValueFromPipeline = true,
+            Position = 0)]
+        [ValidateNotNullOrEmpty]
+        [ArgumentCompleter(typeof(NetworkArgumentCompleter))]
+        public string[] Id { get; set; }
```

Comment:
The docker API does this.  For everything other than GET /<object>/ (be it container or network), ID means either ID or name.  In GET of the object, their api supports filters that actually distinguish between ID and Name.